src/frontend-build.sh

```diff
@@ -4,10 +4,10 @@
 # Export vars for helper scripts to use
 # --------------------------------------------
 export APP_NAME=$(node -e "console.log(require(\"${WORKSPACE:-.}${APP_DIR:-}/package.json\").insights.appname)")
-export CONTAINER_NAME="$APP_NAME-pr-check-$ghprbPullId"
-# main IMAGE var is exported from the pr_check.sh parent file
+export CONTAINER_NAME="$APP_NAME-build-main"
+# export IMAGE="quay.io/cloudservices/$COMPONENT-frontend"
```

Comment:
I think we want to keep the previous comment here. Otherwise looks good

Reply:
I don't evne know how that got there, and what was i thinking...so i'm going to blame my fat fingers ;) thx for catching it